docs/api/screen.md

```diff
@@ -133,7 +133,9 @@ Returns [`Point`](structures/point.md)
 Converts a screen physical point to a screen DIP point.
 The DPI scale is performed relative to the display containing the physical point.
 
-### `screen.dipToScreenPoint(point)` _Windows_
+Not currently supported on Wayland.
```

Comment:
Can we specify the behavior when not supported? For example, just mentioning that it returns the point passed in.